UnitySDK/Assets/ML-Agents/Scripts/DemonstrationStore.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.IO;
+using UnityEngine;
+
+
+namespace MLAgents
+{
+    /// <summary>
+    /// Demonstration meta-data.
+    /// </summary>
+    [System.Serializable]
+    public struct DemonstrationMetaData
+    {
+        public int numberExperiences;
+        public int numberEpisodes;
+        public int API;
+    }
+    
+    public class DemonstrationStore
+    {
+        private string filePath;
+        private DemonstrationMetaData metaData;
+        
+        public void CreateDirectory()
+        {
+            if (!Directory.Exists("Assets/Demonstrations"))
+            {
+                Directory.CreateDirectory("Assets/Demonstrations");
+            }
+        }
+        
+        /// <summary>
+        /// Creates demonstration file, and writes brainParameters as json to file.
+        /// </summary>
+        public void CreateDemonstrationFile(string demonstrationName)
+        {
+            // Creates demonstration file.
+            var literalName = demonstrationName;
```

Comment:
Might make sense to sanitize this string for security

Reply:
Would stripping all characters except alphanumeric work?

Reply:
i think so. the low hanging vulnerability here is demonstration name "../../../../some_important_file_i_dont_want_deleted"